internal/provider/resources/workspace_role_test.go

```diff
@@ -0,0 +1,63 @@
+package resources_test
```

Comment:
just noting that these tests are not yet configured to run in GHA, as there's some extra setup we'll need to do around configuring the test provider to properly inherit env vars.  we'll also want to define a precheck function, which testcases can hook into to ensure all the necessary parameters exist

https://github.com/PrefectHQ/terraform-provider-prefect/issues/30